src/env_loader.zig

```diff
@@ -431,6 +432,55 @@ pub const Loader = struct {
             this.map.get("bamboo.buildKey")) != null;
     }
 
+    pub fn getHttpProxy(this: *Loader, url: URL) ?URL {
+        // TODO: When Web Worker support is added, make sure to intern these strings
+        var http_proxy: ?URL = null;
+
+        if(url.isHTTP()){
+        if (this.map.get("http_proxy")) |proxy| {
```

Comment:
Do you have the zig formatter configured? It's an editor extension you can install tiehaus.zig

Reply:
I will install it and update the files